packages/nc-gui/components/cell/GeoData.vue

```diff
@@ -0,0 +1,154 @@
+<script lang="ts" setup>
+import type { GeoLocationType } from 'nocodb-sdk'
+import { Modal as AModal, useVModel } from '#imports'
+import { latLongToJoinedString } from '~~/utils/geoDataUtils'
```

Comment:
We tend to put utils to `#imports`. What you can do is

- Add `export * from './geoDataUtils'` in  `packages/nc-gui/utils/index.ts`
- then import as `import { latLongToJoinedString } from #imports`. In this case, that would be `import { latLongToJoinedString, Modal as AModal, useVModel } from '#imports'`

Reply:
Thanks!